vispy/io/wavefront.py

```diff
@@ -303,10 +305,13 @@ def writeMesh(self, vertices, faces, normals, values, name=''):
         if faces is None:
             faces = np.arange(len(vertices))
 
-        # Reshape faces
-        Nfaces = faces.size // 3
-        faces = faces.reshape((Nfaces, 3))
-
+        try:
+            # Reshape faces
+            Nfaces = faces.size // 3
+            faces = faces.reshape((Nfaces, 3))
+        except ValueError:
+            logger.warning('''Faces doesn't appear to be triangular, '''
+                           '''be advised the file cannot be read back''')
```

Comment:
Can you please add "... be read back *in Vispy*" here, to make it explicit that this is a vispy thing?